ConsoleApp/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+
+namespace ConsoleApp
+{
+    //i.safontev/homework/14-generics
+    public class Stack<T>
+    {
+        public T[] Arr { get; set; }
+        public int Top { get; set; }
+        public int Count { get; set; }
```

Comment:
i do not understand why do you need it if you never change it, in case if you'll create stack with e.g. 3 elements and when you'll try to push 4th element it will be exception